source/jormungandr/tests/olympic_sites_tests.py

```diff
@@ -168,12 +246,96 @@
 
 @dataset({'main_routing_test': MOCKED_INSTANCE_CONF}, global_config={'activate_bragi': True})
 class TestOlympicSites(AbstractTestFixture):
+    def test_address_with_within_to_address_journeys(self):
+        # forbidden_uris used :  physical_mode:0x0
+        params = {
+            'reverse_timeout': 200,
+            'within_timeout': 200,
+            'pt_dataset[]': 'main_routing_test',
+            'lon': s_lon,
+            'lat': s_lat,
+        }
+        from_place = "{}/multi-reverse?{}".format(BARGI_URL, urlencode(params, doseq=True))
```

Comment:
```suggestion
        from_place = "{}/multi-reverse?{}".format(BRAGI_URL, urlencode(params, doseq=True))
```

And everywhere in the rest of the file, there is the same typo.

Reply:
:+1: